bigchaindb/core.py

```diff
@@ -115,6 +117,18 @@ def info(self, request):
 
         self.abort_if_abci_chain_is_not_synced()
 
+        # Check if BigchainDB supports the Tendermint version
```

Comment:
One more neatpick:

```
if not hasattr(request, 'version') or not tendermint_version_is_compatible(request.version):
    logger.error('Unsupported Tendermint version: {getattr(request, "version", "no version")} ..')
    sys.exit(1)
```

Reply:
A slight correction: `not (hasattr(request, 'version') and tendermint_version_is_compatible(request.version))`